framework/platform/bpl/uci/scripts/platform_daemon_ugw.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+ap_to_repeater() 
+{
+    #configure APM
+    #uci set beerocks.config.management_mode="Proprietary-Mesh"
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"management_mode" : "Proprietary-Mesh"}}'
+    #uci set beerocks.config.operating_mode="WDS-Repeater"
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"operating_mode" : "WDS-Repeater"}}'
+    #uci set beerocks.config.ssid='adam_wave_43'
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"ssid" : "adam_GW_2.4G"}}'
+    #uci set beerocks.config.mode_enabled='None'
+    ubus call uci set '{ "config" : "beerocks" ,"section" : "config", "values" : {"mode_enabled" : "None"}}'
+    #uci commit beerocks
+    ubus call uci commit '{"config" : "beerocks" }'
+    
+    static_init
+    
+    #configure wireless
+    # FH configuration for client connection (optional)
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio13", "values" : {"ssid" : "ssid_RPT_2.4G"}}'
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio13", "values" : {"key_mgmt" : "NONE"}}'
+    #2.4G
+    # 4addr-mode
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio26", "values" : {"wds" : "1"}}'
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio26", "values" : {"vendor_elems" : "13 dd050017353001 14 dd050017353001" } }'
+    # to remove above entry (use 3addr-mode) use: uci delete wireless.default_radio26.vendor_elems 
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio26", "values" : {"full_ch_master_control" : "1"}}'
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio26", "values" : {"disabled" : "0"}}'
+    
+    #5G
+    # 4addr-mode
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio58", "values" : {"wds" : "1"}}'
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio58", "values" : {"vendor_elems" : "13 dd050017353001 14 dd050017353001" } }'
+    
+    ubus call uci set '{ "config" : "wireless", "section" : "default_radio58", "values" : {"full_ch_master_control" : "1"}}'
+    ubus call uci set '{ "config" : "wireless" ,"section" : "default_radio58", "values" : {"disabled" : "0"}}'
+    
+    #uci commit wireless (currently automatically restarts the hostapd's)
+    ubus call uci commit '{"config" : "wireless" }'
+    # If the automatic restard is removed, the follwoing should be used after commit
+    #/etc/init.d/network reload
+}
+repeater_to_ap()
+{
+    # Kill APM
+    /opt/prplMesh/scripts/prplmesh_utils.sh stop
+    
+    #(The daemon shouldn't perform this operation directly - but via UCI)
+    # Kill DHCP client
+	#kill `cat /var/run/udhcpc-br-lan.pid`
+
+    #configure APM
```

Comment:
Not APM. Comment is meaningless, you should write what you are configuring if you already put a comment.

Reply:
removed